cmd/create/oidcprovider/cmd.go

```diff
@@ -143,6 +143,19 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	// Check to see if IAM operator roles have already created
+	missingRoles, err := validateOperatorRoles(awsClient, cluster)
+	if err != nil {
+		reporter.Errorf("Failed to validate operator roles exist: %s", err)
+		os.Exit(1)
+	}
+
+	if len(missingRoles) > 0 {
+		reporter.Errorf("Unable to find all required IAM roles for operators:\n%s\n\n See rosa create operator-roles --help",
+			strings.Join(missingRoles, "\n"))
```

Comment:
This is very nice, thanks!
I will suggest surrounding the command in single quotes: `See 'rosa create operator-roles --help'`